packages/wpcom-proxy-request/src/index.js

```diff
@@ -307,6 +296,9 @@ function install() {
 	// create the <iframe>
 	iframe = document.createElement( 'iframe' );
 
+	const origin = window.location.protocol + '//' + window.location.host;
+	debug( 'using "origin": %o', origin );
```

Comment:
Nowadays we could use `window.location.origin`. We [couldn't](https://caniuse.com/mdn-api_origin) when we supported IE 11, but now all supported browsers support it.

Reply:
Nice, TIL! I'll adjust to use it instead.